ledger/tests/ledger.rs

```diff
@@ -81,3 +81,56 @@ async fn user_buys_and_sells_usd() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn adjust_exchange_position() -> anyhow::Result<()> {
+    let pool = init_pool().await?;
+    let ledger = Ledger::init(&pool).await?;
+
+    let initial_exchange_balance = ledger
+        .balances()
+        .exchange_position_account_balance(OKEX_POSITION_ID)
+        .await?
+        .map(|b| b.settled())
+        .unwrap_or(Decimal::ZERO);
+
+    ledger
```

Comment:
this should not be exposed. Expose `adjust_okex_position` like in the calling code in `poll_okex`